frontend/src/App.js

```diff
@@ -31,37 +23,35 @@ class Business extends React.Component {
       <Reviews reviews={data.reviews} />
     </div>
   );
-  }
-}
-
+};
 
-class Reviews extends React.Component {
-  render() {
-    const reviews = this.props.reviews;
-    const listItems = reviews.map((review) =>
-    <tr>
-      <Review review={review} />
-    </tr>
-  );
+const Reviews = ({reviews}) => {
+  const listItems = reviews.map((review, i) => <Review key={i} review={review} />);
   return (
     <table>
+      <thead>
         <tr>
           <th>Yelp Rating</th>
           <th>Model Rating</th>
           <th>Review</th> 
         </tr>
-      {listItems}
+      </thead>
+      <tbody>
+        {listItems}
+      </tbody>
     </table>
-    
   );
-  }
-}
+};
 
 export default class App extends React.Component {
-  state = { data: null, name: '', reviews: {} };
+  state = {
+    data: null,
+    name: '',
+    reviews: {}
+  };
 
   handleChange = event => {
-    this.setState({ name: event.target.value });
+    this.setState({name: event.target.value});
```

Comment:
we can prolly yank out `name` because an artifact from the tutorial i yanked this out from lol